jre_emul/Classes/NSString+JavaString.m

```diff
@@ -1005,6 +1004,7 @@ + (const J2ObjcClassInfo *)__metadata {
     { "indexOfString:fromIndex:", "I", 0x1, 45, 46, -1, -1, -1, -1 },
     { "intern", "LNSString;", 0x1, -1, -1, -1, -1, -1, -1 },
     { "isEmpty", "Z", 0x1, -1, -1, -1, -1, -1, -1 },
+    { "java_compareTo:", "I", 0x1, 33, 13, -1, -1, -1, -1 },
```

Comment:
Can someone more familiar with the way J2ObjC reflection works verify that changing the ordering here isn't going to throw any indexes off?